src/components/Search/utils.ts

```diff
@@ -37,7 +37,7 @@ export const getURLByAggregateSearchResult = (result: AggregateSearchResult) =>
       if (attributes.udtType === 'xudt') {
         return `/xudt/${attributes.typeHash}`
       }
-      break
+      return `/sudt/${attributes.typeHash}`
```

Comment:
In what case will `udtType` not be `sudt` but the page should be redirected to `sudt page`

Reply:
`spore_cell`, I didn't find the specific page, but I found that the `sudt` page can view all udt (sudt / xudt / spore etc..), so I added this as fallback

Reply:
`Spore` was covered in `nft` page, e.g. https://explorer.nervos.org/nft-info/0x42790e8287162a26801fc14d65c1624524c25dd14ee4212c7b7150e1d4f58df5/38137753267183539873752221734122950207355577885987165145598678283704816207539

Reply:
Please check @PainterPuppets

Reply:
> Please check @PainterPuppets

Got it.

Reply:
> Please check @PainterPuppets

https://github.com/Magickbase/ckb-explorer-public-issues/issues/676#issuecomment-2144362179

I'm a little confused as to why `spore_cell` would be an `udt` type, and then I tried to access `https://explorer.nervos.org/nft-collections/0x43ec3b508b32bc3976a9f7366facd5281e7ba801dff531ee9e4b415152376cbe ` (mentioned in issue Error Search Results) and found that the nft page doesn't show up, I'm wondering if this is a data error or something, is it normal for `spore_cell` to appear in `udt`?

Reply:
> > Please check @PainterPuppets
> 
> [Magickbase/ckb-explorer-public-issues#676 (comment)](https://github.com/Magickbase/ckb-explorer-public-issues/issues/676#issuecomment-2144362179)
> 
> I'm a little confused as to why `spore_cell` would be an `udt` type, and then I tried to access [explorer.nervos.org/nft-collections/0x43ec3b508b32bc3976a9f7366facd5281e7ba801dff531ee9e4b415152376cbe](https://explorer.nervos.org/nft-collections/0x43ec3b508b32bc3976a9f7366facd5281e7ba801dff531ee9e4b415152376cbe) (mentioned in issue Error Search Results) and found that the nft page doesn't show up, I'm wondering if this is a data error or something, is it normal for `spore_cell` to appear in `udt`?

I got it.

`0x0632184fa0aa9363a7f3a9fbfbcf2708d953178ae75111da1e81427bc80fb1ef` is a type hash of `Spore Item Cell`, copied from https://explorer.nervos.org/transaction/0xcfc39752a33ac994ed061d12907228e4249a04a2503bd21d84c236f635d28fe6, it's a `Unicorn itme` and its page is https://explorer.nervos.org/nft-info/0xfe9ed5346d9126582241a0d3d936b47513f3ad13129557c6bb8a16576dd8b8b6/18306823470741201083262782512497888603216366300343015825462981723873045879564

where

collection id is `0xfe9ed5346d9126582241a0d3d936b47513f3ad13129557c6bb8a16576dd8b8b6`
item id is `0x28794b7e2c16ae9b6751042f102d789c31974b35693e8ac804c3c2e20f839b0c`

I think these data are missing in the result
cc @rabbitz

Reply:
PR: https://github.com/nervosnetwork/ckb-explorer/pull/1958

1. staging search spore item type_hash `0xe158964434ec4984aa6f6ea8676de7c5b685032ba52514cccbb837c62da8a418`, the result is 

```ruby
{
    "data": [
        {
            "id": "205338",
            "type": "token_item",
            "attributes": {
                "name": null,
                "icon_url": null,
                "metadata_url": null,
                "status": "normal",
                "token_id": 4217419134417764539785820948464239161488642572777684692133525740984935036370,
                "token_collection": {
                    "standard": "spore",
                    "name": "Cluster name",
                    "description": "Description of the cluster",
                    "icon_url": null,
                    "symbol": null,
                    "sn": "0xbf7a42821163a5940d6cf66042cce7c9d6e6de86c6ea8186a2e642b6532b77b6"
                }
            }
        }
    ]
}
```

2. search token type hash `0xbf7a42821163a5940d6cf66042cce7c9d6e6de86c6ea8186a2e642b6532b77b6` the result is

```ruby
{
    "data": [
        [
            {
                "id": "18965",
                "type": "token_collection",
                "attributes": {
                    "standard": "spore",
                    "name": "Cluster name",
                    "description": "Description of the cluster",
                    "icon_url": null,
                    "symbol": null,
                    "sn": "0xbf7a42821163a5940d6cf66042cce7c9d6e6de86c6ea8186a2e642b6532b77b6"
                }
            }
        ]
    ]
}
```